Makefile

```diff
@@ -0,0 +1,27 @@
+
+cs:
+	./vendor/bin/phpcs --standard=PSR2 --report=emacs --extensions=php --warning-severity=0 src/ tests/PHPSA/
+
+check-src:
+	./bin/phpsa check -vvv ./src
+
+# Rubric, hacking themselves (c) @Kistamushken
+check-fixtures:
+	./bin/phpsa check -vvv ./fixtures
+
+# For local dev
+dev:
+	./bin/phpsa check -vvv ./sandbox
+
+tests-local:
+	./vendor/bin/phpunit -v
+
+# Alias for tests-local
+tests: tests-local
+
+tests-ci:
+	./vendor/bin/phpunit -v --debug --coverage-clover=coverage.clover
+
+test: cs tests-local check-fixtures check-src
```

Comment:
I would like to change "tests" and/or "test" label to running tests-local and after that cs. Without check-src and check-fixtures because those two generate a lot of output in debug mode and probably only break when phpunit fails too. We would still have "ci" label for running all tests.